docs/debugging-s2i.md

```diff
@@ -0,0 +1,49 @@
+Troubleshooting
```

Comment:
Generally somewhere in the end of the document there should be information to reach out us on #openshift-dev channel with necessary information, meaning --loglevel=5, sti version, reproducible data.